packages/picasso/src/AvatarUpload/story/Default.example.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import { AvatarUpload, Container } from '@toptal/picasso'
+
+const useFile = () => {
+  const [uploading, setUploading] = useState<boolean>(false)
+  const [src, setSrc] = useState<string | undefined>()
+
+  const addFile = (acceptedFile: File) => {
+    const reader = new FileReader()
```

Comment:
If people are going to use this as an example its good to handle acceptedFile if it is null. Otherwise, reading the file would throw an error and didnt trigger `onDropRejected` callback
```suggestion
  const addFile = (acceptedFile: File | null) => {
    if (!acceptedFile) {
      return
    }
    const reader = new FileReader()
```